src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The truck is working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The truck stopped working");
```

Comment:
Similar to the `doWork` method, the message 'The truck stopped working' could be more informative. It should clearly indicate that it's the Truck that has stopped working, for example, 'Truck has stopped working.'